### ISSUE TITLE
Unable to edit project properties on multi-screen different DPI system

### ISSUE THREAD

**Issue by mahoekst** [dotnet/project-system#6093]
**Visual Studio Version**:
2019 16.5.4
**Summary**:
When opening a project to change properties I am unable to select other tabs. If I switch off the optimize for DPI in options it works.

**Steps to Reproduce**:

1. https://onedrive.live.com/?id=FD4E1358FA692D29%212200472&cid=FD4E1358FA692D29

2. 

3. 

**Expected Behavior**:

**Actual Behavior**:

**User Impact**:

**Comment by terrajobst** [comment]
Based on [this Twitter conversation](https://twitter.com/oferns/status/1250497620616888323), it setting this option fixed the issue:

![image](https://user-images.githubusercontent.com/5169960/79380845-ddc5f400-7f15-11ea-9a12-2ebf449dddd6.png)

**Comment by mahoekst** [comment]
Yes, that switch made it work again. So still a bug, but with a workaround :)

**Comment by davkean** [comment]
I wonder if we're reading the wrong DPI here, is the non-4k monitor the main display?

**Comment by davkean** [comment]
I think https://github.com/dotnet/project-system/pull/4841 was supposed to fix this for the property pages, but I'm guessing we're still pulling the wrong DPI for the AppDesigner buttons somewhere.

**Comment by mahoekst** [comment]
My main screen is my surface laptop, I have the VS screen on my other screen (display 3). I attached the dxdiag of my system
[DxDiag.txt](https://github.com/dotnet/project-system/files/4484475/DxDiag.txt)

**Comment by mahoekst** [comment]
Interesting enough the tabs do get highlighted when hovering with the mouse, it's just not getting the click event it seems.
